VERSION

```diff
@@ -1 +1 @@
-0.15.0
+0.16.0
```

Comment:
seems the version might be out of order? See here: https://github.com/cyberark/conjur-authn-k8s-client/pull/62/files  we pumped to 1.0.0 and now we are returning to 0.16.0?

Reply:
we didn't bump to 1.0.0 in the end.